woodwork/logical_types.py

```diff
@@ -389,6 +389,11 @@ class IntegerNullable(LogicalType):
     primary_dtype = "Int64"
     standard_tags = {"numeric"}
 
+    def transform(self, series, null_invalid_values=False):
+        if null_invalid_values:
```

Comment:
Could you add a quick docstring to this describing what's going on here (coersion to string, numeric, and then nulling out floats)